src/NServiceBus.Autofac.AcceptanceTests/When_declaring_a_public_property.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace NServiceBus.Autofac.AcceptanceTests
 {
     using System.Threading.Tasks;
-    using NServiceBus.AcceptanceTesting;
+    using AcceptanceTesting;
```

Comment:
Why was this change made? Personally, I prefer the full namespaces to be used, but I'm happy to go with whatever convention has been decided for this repo. However, the declarations should be consistent. Those in the next two lines are still using the full namespaces.